Add the "groups" column.

andravin opened the pull request:
Fixes #319.

TylerYep commented:
Thanks for submitting this PR! This change is looking good. Could you add a new unit test and get the existing tests passing so I can review this further?

andravin commented:
I added a test for the GROUPS column and fixed the existing tests.

There are two commits. The first one just adds the GROUPS column to the end of the ColumnSettings enumeration, causing the GROUPS column to be last in the printed output.

I thought this looked wrong, so I made a second commit that inserts the GROUPS column after the KERNEL_SIZE column.

TylerYep commented:
Thanks! Great work here
